src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,148 @@
+# declaration-block-no-redundant-longhand-properties
+
+Disallow longhand properties that can be combined into one shorthand property.
+
+```css
+a {
+    padding-top: 1px;
+    padding-right: 1px;
+    padding-bottom: 1px;
+    padding-left: 1px; }
+/** ↑
+ *  These longhand properties */
+```
+
+The longhand properties in the example above can be more concisely written as:
+
+```css
+a {
+  padding: 1px;
+}
+```
+
+This rule will only warn if you've used the longhand equivalent of *all* the properties that the shorthand will set. As missing values are replaced by their initial value, it is impossible to allow inheritance of individual properties by omitting them and use shorthand property.
```

Comment:
I'm finding the why part of this confusing. @davidtheclark Any ideas how it could be improved?